src/Microsoft.Azure.SignalR.Serverless.Protocols/IServerlessProtocol.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Buffers;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Azure.SignalR.Serverless.Protocols
```

Comment:
>Microsoft.Azure.SignalR.Serverless.Protocols [](start = 10, length = 44)

Another question, should this be part of Microsoft.Azure.SignalR.Protocols? Could it be used besides Azure Function? Will we publish a new package for this?

Reply:
If internal only, shall we make the interfaces and class internal and make it visible internal?

---
In reply to: [380490961](https://github.com/Azure/azure-functions-signalrservice-extension/pull/92#discussion_r380490961) [](ancestors = 380490961)

Reply:
Yes, it need to be publish as a seperate package. However, I think it doesn't need to be a part of `Microsoft.Azure.SignalR.Protocols`. And we will move it to SDK repo later.

Reply:
It's not only for internal use. People who are not using Azure Function trigger may leverage it directly.